irods/pool.py

```diff
@@ -87,6 +97,9 @@ def get_connection(self):
 
             logger.debug("Adding connection with id {} to active set".format(id(conn)))
 
+            if not new_conn:
+                _adjust_timeout_to_pool_default(conn)
```

Comment:
So the creation of a **new** connection doesn't need to do this because new connections already take the timeout value into account?

Reply:
If I had to guess, the timeout values used by `Connection`s are not updated if the pool of which they are a part is updated... Possibly because the `Connection.pool` member is a copy of the connection pool?

A comment explaining why we have to do this could be helpful in any case. I'll let @d-w-moore weigh in, though. :)

Reply:
@korydraughn  is correct. I will insert comment to be sure. Connections receive a timeout value when they instantiate or modify their internal socket object, and that happens  in one of two ways: either when `Connection._connect` is called (usually from `__init__`) or in this new routine .  `_adjust_timeout_to_pool_default` -- which sets  the conn's internal reference to the owning pool. (I must stress here it's a reference, not a copy. objects are only copied when you see the word copy, (example: `import copy; obj2 = copy.copy(obj1)` ) and only deeply so when you see a call to copy.deepcopy

Reply:
Thanks for the explanation.